whatToDad_site/whatToDad_app/templates/activity.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %} {% block content %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1> Activities </h1>
+    <br>
+    <div class="container">
+        <div class="row">
+            <ul>
+                {% for activity in activities %}
+                <br>
```

Comment:
Only `<ol>` and `<li>` should be direct children of `<ul>`. If you need more space, style the `li` with css.